diff --git a/app/src/main/java/com/particlesdevs/photoncamera/processing/opengl/postpipeline/MotionV2Render.java b/app/src/main/java/com/particlesdevs/photoncamera/processing/opengl/postpipeline/MotionV2Render.java
index 00d08f70a242bd5e8fc2e459c9b3fec67673329b..e3e9d687dac0ba4dbee1b37dd528da9d10fddbf9 100644
--- a/app/src/main/java/com/particlesdevs/photoncamera/processing/opengl/postpipeline/MotionV2Render.java
+++ b/app/src/main/java/com/particlesdevs/photoncamera/processing/opengl/postpipeline/MotionV2Render.java
@@ -1117,7 +1117,7 @@ public final class MotionV2Render extends Node {
             gainMap = new GLTexture(basePipeline.mParameters.mapSize,
                     new GLFormat(GLFormat.DataType.FLOAT_16, 4),
                     BufferUtils.getFrom(basePipeline.mParameters.gainMap), GL_LINEAR, GL_CLAMP_TO_EDGE);
-            GLTexture out = new GLTexture(source.mSize, new GLFormat(GLFormat.DataType.FLOAT_16, 4));
+            GLTexture out = new GLTexture(source.mSize, new GLFormat(GLFormat.DataType.FLOAT_16));
             glProg.useAssetProgram("motionv2/photon_new_prepare");
             glProg.setTexture("InputBuffer", source);
             glProg.setTexture("GainMap", gainMap);
diff --git a/app/version.properties b/app/version.properties
index 182511e67aa0b2b63f8d3c680dd76aca075dd8b1..e49711daace686af150869d030d2f951a1ddb579 100644
--- a/app/version.properties
+++ b/app/version.properties
@@ -1,4 +1,4 @@
 VERSION_MAJOR=0
 VERSION_MINOR=9726440
-VERSION_NAME=0.9726657
-VERSION_BUILD=26657
+VERSION_NAME=0.9726656
+VERSION_BUILD=26656
